docs/Auth-with-Ionic2.md

```diff
@@ -142,7 +142,7 @@ and add the following three entries.
 
 your `app.module.ts` file should look something like this.
```

Comment:
Mind if I do that in a follow up PR? I wanted to actually test the code too and do some reworking here.